docs/translations.example.js

```diff
@@ -20,10 +25,10 @@ const unknownTagFunction = () => ``;
 // Define and export translations here
 // Translations must be sorted by component and then alphabetically
 // Translations must be grouped with fold region comments like this:
-//#region cc-component
-//#endregion
+/**
+ * @type {Translations}
+ */
 export const translations = {
-
   // First, declare the translated language name with some emoji flag with "LANGUAGE"
   LANGUAGE: 'Français',
```

Comment:
issue: I think we decided we would remove the `LANGUAGE` key as we no longer need it and it brings more problems than solutions

:arrow_right: this change is available in my proposition branch: https://github.com/CleverCloud/clever-components/tree/typecheck-i18n--hubert